jymartineau/songhub-next
# CWR + EBR Tokens Minting Integration

# Testing Instructions #

EBR
1. Install Metamask wallet extension if it is not installed
2. Connect MetaMask wallet using "Connect Wallet" Button
3. Import EBR
4. Confirm transaction in Metamask extension
